analyzers/packaging/SonarAnalyzer.CSharp.nuspec

```diff
@@ -22,12 +22,12 @@
     </frameworkAssemblies>
   </metadata>
   <files>
-    <file src="bin\Release\SonarAnalyzer.CFG.dll" target="analyzers" />
-    <file src="bin\Release\SonarAnalyzer.dll" target="analyzers" />
-    <file src="bin\Release\SonarAnalyzer.CSharp.dll" target="analyzers" />
-    <file src="bin\Release\Google.Protobuf.dll" target="analyzers" />
+    <file src="..\src\SonarAnalyzer.CSharp\bin\Release\net46\SonarAnalyzer.CFG.dll" target="analyzers" />
```

Comment:
An idea for future refactoring (not this PR): We can unify `install.ps1` and `uninstall.ps1` for C# and VB.NET